CONTRIBUTING.md

```diff
@@ -215,7 +215,7 @@ style defined by the
 
 C++ Code formatting is managed with a custom ClangFormat configuration file.
 This is the `.clang-format` file found at the base of the repo tree.
-It is intended for use with **ClangFormat version 9.0.1** (see
+It is intended for use with **ClangFormat version 14.0.1** (see
```

Comment:
The run in this commit actually used `14.0.6`
![Screenshot from 2025-03-15 08-12-25](https://github.com/user-attachments/assets/b9ecd186-7e4c-42a9-8801-95bebf053f39)

Reply:
I think specifying the patch version is a bit excessive and probably only going to be out of date as the patch version gets incremented.